Rearrange the tuning profile section (#2666)
theforeman/foreman-documentation: guides/common/modules/proc_configuring-installation.adoc

This can be your company name.
An internal label that matches the value is also created and cannot be changed afterwards.
If you do not specify a value, an organization called *Default Organization* with the label *Default_Organization* is created.
You can rename the organization name but not the label.
* By default, all configuration files configured by the installer are managed by Puppet.
When `{foreman-installer}` runs, it overwrites any manual changes to the Puppet managed files with the initial values.
By default, {ProjectServer} is installed with the Puppet agent running as a service.
If required, you can disable Puppet agent on {ProjectServer} using the `--puppet-runmode=none` option.
* If you want to manage DNS files and DHCP files manually, use the `--foreman-proxy-dns-managed=false` and `--foreman-proxy-dhcp-managed=false` options so that Puppet does not manage the files related to the respective services.
For more information on how to apply custom configuration on other services, see {InstallingServerDocURL}applying-custom-configuration_{project-context}[Applying Custom Configuration to {Project}].

> * If your {Project} deployment includes over 5000 hosts, you can improve its performance by using predefined tuning profiles. For more information, see xref:tuning-with-predefined-profiles_{context}[]

Reviewer: ```suggestion
* If your {Project} deployment includes over 5000 hosts, you can improve its performance by using predefined tuning profiles.
For more information, see xref:tuning-with-predefined-profiles_{context}[]
```

This can be your company name.
An internal label that matches the value is also created and cannot be changed afterwards.
If you do not specify a value, an organization called *Default Organization* with the label *Default_Organization* is created.
You can rename the organization name but not the label.
* By default, all configuration files configured by the installer are managed by Puppet.
When `{foreman-installer}` runs, it overwrites any manual changes to the Puppet managed files with the initial values.
By default, {ProjectServer} is installed with the Puppet agent running as a service.
If required, you can disable Puppet agent on {ProjectServer} using the `--puppet-runmode=none` option.
* If you want to manage DNS files and DHCP files manually, use the `--foreman-proxy-dns-managed=false` and `--foreman-proxy-dhcp-managed=false` options so that Puppet does not manage the files related to the respective services.
For more information on how to apply custom configuration on other services, see {InstallingServerDocURL}applying-custom-configuration_{project-context}[Applying Custom Configuration to {Project}].
ifdef::katello,orcharhino,satellite[]

> * If your {Project} deployment includes over 5000 hosts, you can improve its performance by using predefined tuning profiles.

Reviewer 2: Is it needed to list the exact number? Why not
```suggestion
* For larger deployments you can improve its performance by using predefined tuning profiles.
```

> For more information, see xref:tuning-with-predefined-profiles_{context}[]

Reviewer: ```suggestion
For more information, see xref:tuning-with-predefined-profiles_{context}[].
```
